docs/data/date-pickers/time-range-picker/time-range-picker.md

```diff
@@ -0,0 +1,18 @@
+---
+product: date-pickers
```

Comment:
Why is the product not MUI X?

```suggestion
product: mui-x
```

Reply:
I kept it as is during the migration

https://github.com/mui/material-ui/pull/32178/files#r848633431

I asked the same question here :laughing: